BrewMesModulesParent/common/src/main/java/com/brewmes/common/services/ISubscribeService.java

```diff
@@ -0,0 +1,25 @@
+package com.brewmes.common.services;
+
+
+import com.brewmes.common.entities.MachineData;
+
+public interface ISubscribeService {
+
+    /**
+     * Sets up a subscribtion to the machine values based on a {@code connectionID} representing the machine.
+     * @param connectionID The id of the connection to subscribe to.
+     * @return {@code true} if the subscribtion went successfully, and {@code false} if something went wrong.
+     */
+ boolean subscribeToMachineValues(String connectionID);
+
+    /**
+     * Should return the latest {@code MachineData} measured.
+     * @param connectionID the connectionID representing the machines connection.
+     * @return {@code MachineData} if there exists a current subscribtion on related to the {@code connectionID},
+     * returns {@code null} if no subscribtions were found for the {@code connectionID}.
```

Comment:
```suggestion
     * @return {@code MachineData} if there exists an active subscribtion on the {@code connectionID},
     * returns {@code null} if no subscribtions were found for the {@code connectionID}.
```
Minor grammatical fix.